azure-pipelines-autogen.yml

```diff
@@ -0,0 +1,31 @@
+pr: none
+
+schedules:
+- cron: "0 8 * * *"
+  branches:
+    include:
+    - master
+  displayName: Daily build
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+steps:
+- checkout: self
+  persistCredentials: true
+  clean: true
+- script: |
+    cd generator
+    npm install
+    npm run generate-all
+    cd ..
+  displayName: Generate schemas
+- script: |
+    git config user.name "Autogenerator Pipeline"
+    git checkout -b autogenerate
+    git add --all schemas
+    if ! git diff-index --quiet HEAD --; then
+      git commit -m "Autogenerate schemas"
```

Comment:
>commit [](start = 10, length = 6)

can there be conflicts to resolve?

Reply:
No - it doesn't do any merging/rebasing. It'll just re-run the generation and then create a git commit with the changes.